gatling-http/src/main/scala/io/gatling/http/Predef.scala

```diff
@@ -15,8 +15,32 @@
  */
 package io.gatling.http
 
+import io.gatling.http.check.HttpCheck
+import io.gatling.http.response.Response
+import io.gatling.http.check.HttpCheckScope
+
+import io.gatling.commons.validation.Success
+
+import io.gatling.core.session._
+import io.gatling.core.check.Check
+import io.gatling.core.check.ConditionalCheck._
+
 object Predef extends HttpDsl {
 
   type Request = org.asynchttpclient.Request
   type Response = io.gatling.http.response.Response
+
+  implicit object HttpCheckWrapper extends CheckWrapper[Response, HttpCheck] {
+    def wrap(check: Check[Response]): HttpCheck = new HttpCheck(check, HttpCheckScope.Body, None)
```

Comment:
I knew this was a flaw, but I didn't have the big picture, so I cannot do best. I'll try dig in your way.